docs/content/app-protect/configuration.md

```diff
@@ -1,33 +1,31 @@
 ---
 title: Configuration
 
-description: "This document describes how to configure the NGINX App Protect module."
+description: "This document describes how to configure the NGINX App Protect WAF module."
```

Comment:
Should the folder here also be `app-protect-waf` instead of just `app-protect`? Also the folder in `ingress-resources`?

I've also noticed that in the docs is `app-protect` but in the examples is `appprotect`.